integration-tests/tests/src/tests/sync/client-reset.ts

```diff
@@ -44,26 +44,26 @@ function getPartitionValue() {
   return new UUID().toHexString();
 }
 
-async function triggerClientReset(app: Realm.App, user: Realm.User): Promise<void> {
+async function triggerClientReset(app: App, user: User): Promise<void> {
   await user.functions.triggerClientReset(app.id, user.id);
 }
 
 async function waitServerSideClientResetDiscardUnsyncedChangesCallbacks(
   useFlexibleSync: boolean,
   schema: Realm.ObjectSchema[],
-  app: Realm.App,
-  user: Realm.User,
+  app: App,
+  user: User,
   actionBefore: (realm: Realm) => void,
   actionAfter: (beforeRealm: Realm, afterRealm: Realm) => void,
 ): Promise<void> {
   const resetHandle = createPromiseHandle();
   let afterCalled = false;
   let beforeCalled = false;
-
-  const realm = new Realm({
+  const config: Configuration = {
     schema,
     sync: {
       user,
+      // @ts-expect-error this setting is not for users to consume
```

Comment:
is there any other recommended way to treat this ts-error?

Reply:
I don't think so. `_sessionStopPolicy` is only for our internal testing, and we don't wish our users to use it.